src/Tools/BuildValidator/BuildValidator.csproj

```diff
@@ -4,7 +4,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```

Comment:
This is done to match the ordering of `TargetFrameworks` in the csc / vbc project files. This way when F5 between them and BuildValidator locally you're using the same runtime. Important for inner dev loop.